pkg/controller/datavolume-controller.go

```diff
@@ -53,10 +53,52 @@ const (
 	SuccessSynced = "Synced"
 	// ErrResourceExists provides a const to indicate a resource exists error
 	ErrResourceExists = "ErrResourceExists"
+	// ErrResourceDoesntExist provides a const to indicate a resource doesn't exist error
+	ErrResourceDoesntExist = "ErrResourceDoesntExist"
+	// ErrClaimLost provides a const to indicate a claim is lost
+	ErrClaimLost = "ErrClaimLost"
+	// DataVolumeFailed provides a const to represent DataVolume failed status
+	DataVolumeFailed = "DataVolumeFailed"
+	// ImportScheduled provides a const to indicate import is scheduled
+	ImportScheduled = "ImportScheduled"
+	// ImportInProgress provides a const to indicate an import is in progress
+	ImportInProgress = "ImportInProgress"
+	// ImportFailed provides a const to indicate import has failed
+	ImportFailed = "ImportFailed"
+	// ImportSucceeded provides a const to indicate import has succeeded
+	ImportSucceeded = "ImportSucceded"
+	// CloneScheduled provides a const to indicate clone is scheduled
+	CloneScheduled = "CloneScheduled"
+	// CloneInProgress provides a const to indicate clone is in progress
+	CloneInProgress = "CloneInProgress"
+	// CloneFailed provides a const to indicate clone has failed
+	CloneFailed = "CloneFailed"
+	// CloneSucceeded provides a const to indicate clone has succeeded
+	CloneSucceeded = "CloneSucceeded"
 	// MessageResourceExists provides a const to form a resource exists error message
 	MessageResourceExists = "Resource %q already exists and is not managed by DataVolume"
+	// MessageResourceDoesntExist provides a const to form a resource doesn't exist error message
+	MessageResourceDoesntExist = "Resource managed by %q doesn't exist"
 	// MessageResourceSynced provides a const to standardize a Resource Synced message
 	MessageResourceSynced = "DataVolume synced successfully"
+	// MessageErrClaimLost provides a const to form claim lost message
+	MessageErrClaimLost = "PVC %q lost"
+	// MessageImportScheduled provides a const to form import is scheduled message
+	MessageImportScheduled = "Import %q scheduled"
```

Comment:
I would say "Import into %s scheduled."

%q is used when you want to escape or quote parts of the string like if you are displaying JSON. Since this will be a basic string %s is more appropriate. The 'into' is also important as otherwise it seems an import named the PVC name is scheduled which is not the case, we are importing into the PVC.